packages/cli/test/unit/commands/domains/add.test.ts

```diff
@@ -1,9 +1,66 @@
-import { describe } from 'vitest';
+import { describe, it, expect } from 'vitest';
+import domains from '../../../../src/commands/domains';
+import { client } from '../../../mocks/client';
+import { useDomain } from '../../../mocks/domains';
+import { useProject } from '../../../mocks/project';
+import { useUser } from '../../../mocks/user';
 
-describe.todo('domains add', () => {
-  describe.todo('[name]', () => {
-    describe.todo('[project]', () => {
-      describe.todo('--force');
+describe('domains add', () => {
+  describe('[name]', () => {
+    describe('[project]', () => {
+      describe('--force', () => {
+        it('tracks telemetry data, async', async () => {
```

Comment:
What does "async" mean here?

Reply:
I copied that from another `vercel domains` telemetry PR that @trek wrote: https://github.com/vercel/vercel/blob/d1743d1757f51ff94f72b714ff048584c9abcd31/packages/cli/test/unit/commands/domains/ls.test.ts#L43

Reply:
Yeah, that was just some moving-too-fast-in-my-editor junk

Reply:
Approved to keep us going but @erikareads or I should swing back and fix both of these after.